crates/subspace-core-primitives/src/lib.rs

```diff
@@ -83,6 +83,48 @@ pub const BLAKE3_HASH_SIZE: usize = 32;
 /// BLAKE3 hash output
 pub type Blake3Hash = [u8; BLAKE3_HASH_SIZE];
 
+/// 1 in 6 slots (on average, not counting collisions) will have a block.
+/// Must match ratio between block and slot duration in constants above.
+const SLOT_PROBABILITY: (u64, u64) = (1, 6);
+
+/// How many pieces one sector is supposed to contain (max)
+const MAX_PIECES_IN_SECTOR: u16 = 1000;
```

Comment:
These constants should be removed and made explicit arguments in below functions